api/models/process.go

```diff
@@ -247,32 +249,32 @@ func ListProcesses(app string) ([]*Process, error) {
 		select {
 		case ps := <-psch:
 			pss = append(pss, &ps)
-		case err := <-errch:
-			return nil, err
 		}
 	}
 
 	pending, err := ListPendingProcesses(app)
-
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get pending processes: %s\"\n", err)
```

Comment:
Are there particular errors that you found down this path? As this is a pure ECS call, I hope that it's fairly reliable.

I also think that surfacing pending processes during a deploy is important visibility... Though I'm not sure if a total error would be any better than a process list without pending processes.